### DISCUSSION TITLE
New data source: Appwrite 🎀

### DISCUSSION THREAD

**Discussion by Meldiron** [chartbrew/chartbrew#157] [Ideas]
[Appwrite](https://appwrite.io/) is a backend as a service, similar to Supabase or Firebase, and has over 25k stars on GitHub. I would love to see an adapter for it in Chartbrew.

**Comment by razvanilin** [comment]
Hey @Meldiron good suggestion! Not sure exactly when I'll have time to work on this though. If there's enough interest I can prioritize it.

Meanwhile, you can use the REST API to fetch data from Appwrite: https://appwrite.io/docs/server/users
